kuma/install.go

```diff
@@ -17,22 +17,23 @@ type MeshInstance struct {
 }
 
 // CreateInstance installs and creates a mesh environment up and running
-func (h *handler) installKuma(del bool) (string, error) {
+func (h *handler) installKuma(del bool, version string) (string, error) {
 	status := "installing"
 
 	if del {
 		status = "removing"
-		// Implement delete instance
 	}
 
-	meshinstance := &MeshInstance{}
+	meshinstance := &MeshInstance{
+		InstallVersion: version,
+	}
 	err := h.config.MeshInstance(meshinstance)
 	if err != nil {
 		return status, ErrMeshConfig(err)
 	}
 
 	h.log.Info("Installing Kuma")
-	err = meshinstance.installKumactl()
+	err = meshinstance.installKumactl(del)
```

Comment:
Could this be changed to `installUsingKumactl`?
I found it a bit confusing that an `install`-func has a `del` parameter. 
Also, could `del` be changed to `delete`?

Reply:
@mgfeller Yes agreed. Changing this now.

Reply:
Also delete is a keyword that means something in golang, thus had to keep it as del. And install command has del because of the current protobuf structure. This can be removed and restructured easily later on.

Reply:
I suspected that `delete` is  a keyword, but couldn't see it here https://golang.org/ref/spec#Keywords and replaced and didn't get a complaint. Probably didn't test thoroughly. 
Thanks for clearing this up :-)